### ISSUE TITLE
Features Request: Commentary view enhancement suggestions

### ISSUE THREAD

**Issue by tianmiaohu** [crosswire/xiphos#940]
Currently the commentary view is auto-linked to the Bible view.  In other words, when the user clicks a verse, the commentary will jump the content to that verse.  This is rather restrictive and sometimes disrupts the user experience.  My suggestions are:
1. Make the commentary view similar to the Bible view.  Allow multiple top tabs so that you can have multiple commentaries open at the same time for the user to choose.
2. Do not auto-link the commentary to the Bible verse.  Instead, provide a Link button so that the user can link if needed.
3. Save the user's current viewing position.  This is very helpful for a length commentary section.  If the user closes Xiphos and reopens, the user should be presented with the last commentary viewing position.  This way the user does not need to search for his last reading position.  This suggestion is applicable to the Book view as well.

Some of the above suggestions are already present in the And Bible.  I believe once implemented, it will give a great user experience.

I really appreciate your creation of a great Bible study tool.  May God bless you and your works!

**Comment by johndudeck** [comment]
How about the approach used by Logos: Link groups (A, B, C, D, and E). You can assign resources to a group, and when you scroll one resource, others in the link group will scroll with it. Assigning a resource to a group is done by clicking on the info icon for that resource and choosing one of the five group buttons on the info drop-down. Presumably only Bibles and Commentaries have this feature available.

As for the viewing position, when leaving Logos, it saves your desktop layout with your location in each open resource in a list, and when you come back, it restores the layout, or you can choose other previous layouts from the layout list. You can also mark one of the layouts as your default that it will open when starting up, rather than the most recent layout.

**Comment by alerque** [comment]
Linking is sometimes quite handy, sometimes quite obnoxious. I've even found myself copying bits of commentary out to another app just because I couldn't easily get back to it as I keep navigating around the Bible. An option to unlink the panes would be great, and remembering the last scroll position when re-opening a pane would be great too.

@johndudeck Linking groups sounds like a nice way to implement this too. Things you didn't want linked could just be in a group by themselves.

**Comment by LAfricain** [comment]
I think this is already possible. Right click on the Commentaries → Open in new tab or new windows. Scroll as you want.

**Comment by karlkleinpaste** [comment]
Yes, separate windows need not link with main window navigation. There is a synchronize button on separate windows, to make the auto-track-with-main-window choice.

As for "multiple top tabs," that's already possible, and kind of normal: Open a new tab with the "+" button, then in that tab you can disable visibility of everything but the commentary. Turn off tab linkage via the View menu, so that tabs navigate independently.

The only other aspect of this is remembering scroll location within a pane, which I'm sorry to say seems not to be possible given the structure of the windows -- there is no way to keep track of e.g. the percentage down into the module the window scroll point is currently displaying.

Xiphos remembers all the basic windows' presence, location, and geometry (main, adv.search, mod.mgr, prefs) including navigation in each main window tab, but does not also remember separate windows.

Unlinking each main window tab's bible+commentary seems not a good idea. The commentary (when done by chapter, which is default) is displayed with a marker line at the current verse's content, specifically placed there to help the user keep track of where he is, so that in the midst of scrolling around, he can easily visually return to where the section started.

Essentially, everything suggested appears to be available, with the exception of scroll fraction in displayed panes.

**Comment by karlkleinpaste** [comment]
The sync button on separate bible/commentary windows:

![Image](https://github.com/user-attachments/assets/0d3d42b1-d0a1-41a1-b8ce-317b102356c3)
